tools/taskcluster/import_strings_and_create_pull_request.sh

```diff
@@ -0,0 +1,32 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This script imports the latest strings, creates a commit, pushes
+# it to the bot's repository and creates a pull request.
+
+# If a command fails then do not proceed and fail this script too.
+set -e
+
+# Go to project root
+cd "$(dirname "$0")"
+cd ../..
+
+# Update local checkout
+git fetch origin
+git reset --mixed origin/master
```

Comment:
Is this --mixed because there is other development going on in the local user master?

Reply:
I used "--hard" first and somehow couldn't figure out why everything behaved really weirdly. Then I noticed that this will also reset the state of the scripts I'm currently writing and testing. To prevent this I switched to "--mixed" which will not reset the local changes :D